paper-ripple-behavior.html

```diff
@@ -77,12 +77,14 @@
         if (rippleContainer) {
           Polymer.dom(rippleContainer).appendChild(this._ripple);
         }
-        var domContainer = rippleContainer === this.shadyRoot ? this :
-          rippleContainer;
-        if (opt_triggeringEvent) {
-          var target = opt_triggeringEvent.target;
-          if (domContainer.contains(/** @type {Node} */(target))) {
-            this._ripple.uiDownAction(opt_triggeringEvent);
+        if (optTriggeringEvent) {
+          // Check if the event happened inside of the ripple container
+          // Fall back to host instead of the root because distributed text
+          // nodes are not valid event targets
+          var domContainer = Polymer.dom(this._rippleContainer || this);
+          var target = Polymer.dom(optTriggeringEvent).rootTarget;
+          if (domContainer.deepContains( /** @type {Node} */(target))) {
```

Comment:
After last update when clicked on a radio or a checkbox gives: Uncaught TypeError: domContainer.deepContains is not a function
Polymer.PaperRippleBehavior.ensureRipple	@	paper-ripple-behavior.html:86
Polymer.PaperRippleBehavior._downHandler	@	paper-ripple-behavior.html:62
handler	@	polymer.html:402
Polymer.Base._addFeature.fire	@	polymer.html:1209
Gestures.fire	@	polymer.html:733
Gestures.register.fire	@	polymer.html:811
Gestures.register.mousedown	@	polymer.html:801
Gestures.handleNative	@	polymer.html:625